### PR TITLE
Added new resource

### PR DESCRIPTION
**Pull request by Johnniexson**

### CODE REVIEW

**Comment by timdeschryver** [comment]
Hi thanks for the contribution, it seems like there is a conflict.
Could you resolve this?

**Comment by Johnniexson** [comment]
Done

**Comment by Johnniexson** [comment]
What about nw?

File: src/assets/resources.json [hunk 1]
```diff
@@ -127,6 +127,20 @@
       }
     },
     {
+      "title": "Going full-stack Angular with NestJs",
+      "description": "A demo showing how to add NestJS to an existing Angular application and take advantage of their tooling to show off Server Side Rendering with a fully integrated frontend and backend. (Harnessing the power of Typescript)",
+      "author": {
+        "name": "Florian Tischler",
+        "profile": "https://twitter.com/floriantischler"
```
Comment (timdeschryver) [inline src/assets/resources.json]: "profile": "https://twitter.com/stephenfluin"

File: src/assets/resources.json [hunk 2]
```diff
@@ -127,6 +127,20 @@
       }
     },
     {
+      "title": "Going full-stack Angular with NestJs",
+      "description": "A demo showing how to add NestJS to an existing Angular application and take advantage of their tooling to show off Server Side Rendering with a fully integrated frontend and backend. (Harnessing the power of Typescript)",
+      "author": {
+        "name": "Florian Tischler",
```
Comment (timdeschryver) [inline src/assets/resources.json]:
```suggestion
        "name": "Stephen Fluin",
```

File: src/assets/resources.json [hunk 3]
```diff
@@ -127,6 +127,20 @@
...
+        "profile": "https://twitter.com/floriantischler"
+      },
+      "url": "https://m.youtube.com/watch?v=y24fC9Pqr8I&t=6s",
+      "category": "library",
```
Comment (timdeschryver) [inline src/assets/resources.json]:
```suggestion
      "category": "recording",
```

**Comment by Johnniexson** [comment]
Great, thanks for letting me know

**Comment by timdeschryver** [comment]
Thanks for your contribution @Johnniexson  🎉
Once the master build is ready it will become available at https://timdeschryver.github.io/hacktoberfest-2019-angular/

**Comment by Johnniexson** [comment]
I look forward to more of this.✌

**Pull request merged**